src/url-params.ts

```diff
@@ -121,6 +149,26 @@ export class AnalyticsUrlParams {
 		return `${URL_PARAM_DEVICE_ID}=${encodeURIComponent(ids.join(','))}`;
 	}
 
+	/**
+	 * @return part of the session ID query parameter string that can be appended to URLs
+	 */
+	getSessionIdQueryString(): string {
+		const id = this.getSessionId();
+		if (id === null) {
```

Comment:
```suggestion
		if (id == null) {
```

So this returns true when checking `undefined == null`.
(in case we go ahead with removing the `null` from the type)